### DISCUSSION TITLE
can't open socket connection to device

### DISCUSSION THREAD

**Discussion by ayal** [TomerFi/aioswitcher#800] [General]
I tried to use the remote switcher server ip for initing `SwitcherType2Api` (i.e `47.91.75.11`), this didn't work

immediately after that I changed the code back to use the local network device ip, but for whatever reason - now I can't even establish the initial socket connection with the device (i.e `open_connection`)

I ended up with zero code changes but now nothing works, everything worked perfectly one minute before trying that :(

Using `discover_devices` works fine, I see the device. I can control the device with my iphone app.

Does this ring any bells? are you aware of some kind of blacklist mechanism or something of this sort? could it be that i messed something up permanently?

**Comment by ayal** [comment]
tried from another pc on the network and it works :/

**Comment by TomerFi** [comment]
This project was built around local communication. A remote switcher server, whatever that is, was never in scope.
